server/models/address.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import "time"
+
+type Address struct {
+	Address                      string    `json:"address" bson:"address"`
+	Owner                        string    `json:"owner" bson:"owner"`
+	Balance                      string    `json:"balance" bson:"balance"`
```

Comment:
These values look like large integers now. Were the units changed from `GO` to `wei`? We should remain compatible with the old api, or use a new service path if we need to change types.

Reply:
I think dealing with the integers is cleaner though, and that is good to have in an additional new field.